.github/workflows/generate_matrix_page.yaml

```diff
@@ -32,8 +32,12 @@ jobs:
     steps:
       - name: Set dynamic env vars
         run: |
+          # GPU Operator dashboard paths
           echo "DASHBOARD_DATA_FILEPATH=${DASHBOARD_OUTPUT_DIR}/gpu_operator_matrix.json" >> "$GITHUB_ENV"
           echo "DASHBOARD_HTML_FILEPATH=${DASHBOARD_OUTPUT_DIR}/gpu_operator_matrix.html" >> "$GITHUB_ENV"
+          # Network Operator dashboard paths
```

Comment:
Why do we want to have the NNO dashboard updated in the same workflow as the GPU operator?  But even then, I think it would be better to separate the NNO steps from the GPU ones. E.g. "Set GPU operator env vars", "Set NNO env vars", "Fetch GPU operator CI results", "Fetch NNO CI resutls", etc.